i3expod.py

```diff
@@ -46,6 +46,18 @@ def get_primary_output_name():
     return None
 
 
+def get_primary_output_monitor_size():
+    xrandr_query = f"xrandr --query | awk -F '[ +]' '/{get_primary_output_name()}/{{print $4}}'"
+    stdout,stderr = subprocess.Popen(xrandr_query, shell=True,
+                    stdout=subprocess.PIPE).communicate()
+
+    # The stdout from awk needs to be decoded, stripped from newline and split
+    # The output should be similar to: b'1920x1080\n', we want a list instead
+    monitor_size = list(map(int, stdout.decode('utf-8').strip().split('x')))
```

Comment:
This one gives me trouble because the result on my system is `['1920', '1080\nleft\nleft\nleft\nleft\nleft\nleft']`
(the split at `x` went great, but the strip only remove the last `\n`)
But you could easily fix by chaning this line to:

```
out = stdout.decode('utf-8')
monitor_size = list(map(int, out[:out.find("\n")].split("x")))
```
This produces the expected result on my system as it cuts the string at the first `\n`. Try it out if it works for you as well but I don't see why it shouldn't :)

Reply:
Great! Fixed it with [ac3b2f1](https://github.com/morrolinux/i3expo-ng/pull/16/commits/ac3b2f1dbfc47a88c150a1189ecfdd87a12e0b17)